src/System.Runtime.Extensions/src/System/IO/Path.Unix.cs

```diff
@@ -183,15 +183,12 @@ public static bool IsPathRooted(string path)
             return path.Length > 0 && path[0] == DirectorySeparatorChar;
         }
 
-        private static unsafe byte[] CreateCryptoRandomByteArray(int byteLength)
+        private static byte[] CreateCryptoRandomByteArray(int byteLength)
         {
             var arr = new byte[byteLength];
-            fixed (byte* buf = arr)
+            if (!Interop.Crypto.GetRandomBytes(arr, arr.Length))
```

Comment:
As long as we're sure that RAND_bytes won't fail due to lack of entropy (and won't block for arbitrary periods of time waiting for, say, mouse movement, like /dev/random often does), I'm happy with the change.